app/middlewares/sessions.js

```diff
@@ -0,0 +1,21 @@
+const { verifyToken } = require('../services/sessions');
+const errors = require('../errors');
+const { INVALID_TOKEN } = require('../../config/constants');
+
+exports.authenticateSession = async (req, res, next) => {
+  try {
+    const bearerHeader = req.headers && req.headers.authorization;
+    if (bearerHeader) {
+      const token = bearerHeader.split(' ')[1];
+      const tokenMetaData = await verifyToken(token);
+      if (tokenMetaData) {
+        // eslint-disable-next-line
+        req.tokenMetaData = tokenMetaData;
```

Comment:
why you disable this line for linter? maybe  `req["tokenMetaData"] = tokenMetaData;` works? 
you could modify linter config file to accept this format, or change you code style, because in the future you're going to have similar lines like this, and you will write `disable-next-line` everywhere

Reply:
Ok, I found a way to bypass that lint validation using `Object.assing`.